egs2/TEMPLATE/asr1/pyscripts/utils/score_lang_id.py

```diff
@@ -0,0 +1,60 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+
+# Copyright 2021 Johns Hopkins University (Jiatong Shi)
+#  Apache 2.0  (http://www.apache.org/licenses/LICENSE-2.0)
+
+import argparse
+import codecs
+import sys
+
+
```

Comment:
This should have been a sym link. I think creating a sym link in windows just copies the file over instead of creating an actual link. Will use a unix system to fix.